### DISCUSSION TITLE
Mistake in README.md: tiddlywiki not available in arch official repo

### DISCUSSION THREAD

**Discussion by foodornt** [TiddlyWiki/TiddlyWiki5#7027] [General]
In the README.md file, it tells arch users to install tiddlywiki through pacman, though it's not available in the main arch repos, it rather exists in the AUR

it should rather be: `yay -S tiddlywiki` (most arch users use yay to manage aur packages)

**Comment by Jermolene** [comment]
Thank you @foodornt – I've updated the instructions in ef779c11e8672cce76ff63070effa2597251540a, which should be live on tiddlywiki.com/github.com in a few minutes.
